### ISSUE TITLE
Macro Error

### ISSUE THREAD

**Issue by svetmymirror** [hannekeo/Worm-align#1]
Hi there, 

I am stuck at the step of identifying the width of the worm, as when I click OK, Fiji gives me "macro error", saying:
 ";" expected in line 55
l = line_width <*> 0.8 ;

How could I resolve this?

Best,
Sveta

**Comment by hannekeo** [comment]
Hi Sveta
Thanks for alerting me to this issue.
It appears to be an issue that FIJI has with the order of line_width * 0.8 as I have reversed it and the macro works now.
You have two options:

-          Either you correct line 55 -like I did- into:



***@***.***D74753.E661B660]



-          Or you download the new version that I have just uploaded to GitHub

You may also find that the generation of the mask does not work optimally in your images in which case you can tweak this part in the macro to work better on your data (i.e. lines 196-200). This is only relevant if you wanted to use the masks in CellProfiler to quantify intensity signals, and not for generation of montages.

Worm_align in the first instance was meant to just be a simple C.Elegans montage maker, but “morphed” into a way to do analysis as well.
Many thanks for using Worm_align! It is great to see my macro being used, and I hope it is helpful in your research.

Kind regards
Hanneke

From: svetmymirror ***@***.***>
Sent: 12 May 2021 16:43
To: hannekeo/Worm-align ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [hannekeo/Worm-align] Macro Error (#1)


Hi there,

I am stuck at the step of identifying the width of the worm, as when I click OK, Fiji gives me "macro error", saying:
";" expected in line 55
l = line_width <*> 0.8 ;

How could I resolve this?

Best,
Sveta

**Comment by hello-lei** [comment]
Yes, it does work. Could I combine some photoes in a group into one picture then to analyst it?

**Comment by hello-lei** [comment]
Hi,the picture of the align is  fuzziness.Can it be clear?

**Comment by berk1835** [comment]
Hi Lei,
What's your image resolution like? If the original image is quite zoomed out / unclear (fuzzy) up close, using functions like 'sharpen' may be able to help after you have finished with Worm Align.


---

Rebekah J White (she/her)

Evolutionary Biology PhD Student

College of Life and Environmental Sciences, Streatham Campus, University of Exeter

Twitter @rebekah_jwhite<https://twitter.com/rebekah_jwhite>


Latest paper: Anthropogenic land-use change and emerging zoonotic diseases<https://onlinelibrary.wiley.com/doi/epdf/10.1111/mam.12201>




________________________________

**Comment by hello-lei** [comment]
Hi,my image resolution is 1280*720.And  does the 'sharpen'  apply to the pictures ot Worm align.
